python/ray/train/v2/tests/conftest.py

```diff
@@ -34,3 +35,26 @@ def shutdown_only():
 def disable_state_actor_polling(monkeypatch):
     monkeypatch.setenv(ENABLE_STATE_ACTOR_RECONCILIATION_ENV_VAR, "0")
     yield
+
+
+@pytest.fixture(autouse=True)
```

Comment:
is it easy to use autouse this only in the places where we need it? I think it's just test_controller and test_worker_group